src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java

```diff
@@ -59,7 +59,6 @@ public static boolean isEmpty(BlockState state) {
     }
 
     @Override
-    @SuppressWarnings("deprecation")
```

Comment:
Why'd you remove this?

Reply:
There was no deprecation warning with it removed